pom.xml

```diff
@@ -77,7 +77,7 @@
     <jackson.version>1.9.11</jackson.version>
     <jackson.package>org.codehaus.jackson</jackson.package>
     <shade.prefix>shaded.parquet</shade.prefix>
-    <hadoop.version>1.1.0</hadoop.version>
+    <hadoop.version>2.3.0</hadoop.version>
```

Comment:
Move the version back to 1.1. Had to do a little more reflection based refactoring to remove references to DirectDecompressors.